node/pkg/driver/device_connectivity/device_path_amd64.go

```diff
@@ -0,0 +1,4 @@
+package device_connectivity
+
+// for x86, the FC subsystem prefix is pci
```

Comment:
I like comments but here it just says exactly as the code line itself

Reply:
then why do we want to keep the comment in Power?

Reply:
I would remove it in power too.

Reply:
done